src/SwaggerProvider.DesignTime/OperationCompiler.fs

```diff
@@ -31,16 +37,19 @@ type OperationCompiler (schema:SwaggerObject, defCompiler:DefinitionCompiler) =
                     let paramDefaultValue = defCompiler.GetDefaultValue x.Type
                     ProvidedParameter(paramName, paramType, false, paramDefaultValue)
             ]
-        let retTy =
+        let retTy, isReturnFile =
             let okResponse = // BUG :  wrong selector
                 op.Responses |> Array.tryFind (fun (code, resp) ->
                     (code.IsSome && (code.Value = 200 || code.Value = 201)) || code.IsNone)
             match okResponse with
             | Some (_,resp) ->
                 match resp.Schema with
-                | None -> typeof<unit>
-                | Some ty -> defCompiler.CompileTy methodName "Response" ty true
-            | None -> typeof<unit>
+                | None -> typeof<unit>, false
+                | Some ty when ty = SchemaObject.File -> 
```

Comment:
this is sort of a hack because I didn't yet want to look at making the definitions compiler learn that the same swagger type (in this case File), could have different representations on the input and output side.